docs/catalogue/catalogue.rst

```diff
@@ -0,0 +1,99 @@
+.. _catalogue:
+
+Catalogue of Past Projects
+==========================
+
+The conversion gallery gives simple examples of conversions from simple combinations of formats.
+
+Some neurophysiology experiments require more diverse combinations of interfaces as well as more exotic functionalities to be NWB and DANDI compatible.
+
+This catalogue contains references and short descriptions for a number of real examples of pipelines used by labs to generate NWB files, many of which are now published on DANDI!
+
+If it sounds as if any of the projects in this catalogue might be similar to your own experiments, it might be worth checking out how they implemented things and asking questions by raising Issues on the respective repositories.
+
+..note: It is likely that each project pinned itself to a specific minor version of NeuroConv, or its predecessor, nwb-conversion-tools (NCT). We have organized each pipeline according to the version used in descending order (newest version first).
```

Comment:
I think the note at the top is not rendering correctly:
<img width="1414" alt="Screenshot 2023-02-21 at 15 23 21" src="https://user-images.githubusercontent.com/24475788/220371391-20d765ff-cc51-4ffb-9571-33ab8898c72c.png">
otherwise looks good to me! Great to see so many projects together.